### PR TITLE
Add License Metadata

### PR DESCRIPTION
**Pull request by ciaranRoche**

### CODE REVIEW

**Comment by ciaranRoche** [comment]
@lgriffin

File: licenses/licenses.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <project>saml-ios-app</project>
+    <version>0.0.0</version>
+    <license>Apache-2.0</license>
+    <dependencies>
+        <dependency>
+            <packageName>fh-ios-sdk</packageName>
+            <version>5.0.0</version>
+            <licenses>
+                <license>
+                    <name>BSD 2-clause \"Simplified\" License</name>
```
Comment (dimitraz) [inline licenses/licenses.xml]: Where did you find this?

Comment (ciaranRoche) [reply licenses/licenses.xml]: here [fh-ios-sdk](https://github.com/feedhenry/fh-ios-sdk)

Comment (dimitraz) [reply licenses/licenses.xml]: Ok, shouldn't it be Apache 2.0. though

Comment (lgriffin) [reply licenses/licenses.xml]: yup it is Apache 2: https://github.com/feedhenry/fh-ios-sdk/blob/master/LICENSE

**Review by lgriffin — APPROVED** [review]
Feel free to merge yourself @ciaranRoche once you update the fh-ios-sdk reference to Apache2

**Comment by lgriffin** [comment]
Actually as Travis is failing / timing out just ping me on it again as I can merge as an admin.

**Comment by ciaranRoche** [comment]
@lgriffin removed bsd and replaced with Apache2

**Pull request merged**